docs/manual/installation.rst

```diff
@@ -1,23 +1,53 @@
 Installation
 ============
 
-**phpspec** is a php 5.3+ library that you'll have in your project
-development environment. Before you begin, ensure that you have at least
-PHP 5.3.3 installed.
+**phpspec** is a php 5.6+ library that you'll have in your project
+development environment. Before you begin, ensure that you have
+PHP 5.6 or 7 installed.
 
-Method #1 (Composer)
---------------------
+Installation process:
+---------------------
 
-The simplest way to install phpspec with all its dependencies is through
-Composer.
+You can install phpspec with all its dependencies through Composer. Follow
+instructions on `the composer website <https://getcomposer.org/download/>`_ if
+you don't have it installed yet.
 
-First, create a ``composer.json`` file in your project's root directory:
+N.b.: You will need to ensure that your Composer ``autoload`` settings are
+correct. phpspec will not be able to detect classes, even ones it has created,
+unless this is working. This is a common issue which causes confusion when
+installing phpspec.
+
+The ``autoload`` section of your ``composer.json`` file may look something like
+this:
+
+.. code-block:: json
+
+    "autoload": {
+        "psr-0": {
+            "": "src/"
```

Comment:
I added it because I was asked to by another contributor (https://github.com/phpspec/phpspec/pull/919#discussion_r64147357), but I don't believe it makes any difference either way.  I'll make sure the two things are consistent.